carbon-design-system/carbon-addons-iot-react
# [ListCard / SimpleList] bug fixes

Closes #1689
Closes #1690 
Closes #1691 

**Summary**

- Adds empty state for ListCard if there is no data
- Merges `i18n` prop with defaultProps `i18n`
- Adds ability to hide the pagination in SimpleList

**Change List (commits, features, bugs, etc)**

- Added story for empty state ListCard
- Added story for hidden pagination in SimpleList

**Acceptance Test (how to verify the PR)**

- Go to ListCard - empty story to see empty state
- Go to SimpleList - hidden pagination story to see hidden pagination AND there is only 1 `i18n` prop being passed here so this also verified the merged i18n
